### ISSUE TITLE
[develop] unable to replay saved cmdlog - row ids not recognized

### ISSUE THREAD

**Issue by vapniks** [saulpw/visidata#422]
When I try to replay a command log I get an error about "no キtable_3 row"
When I delete a sheet, the sheet name is saved with a "キ" prefix char in the command log, and visidata v2.-2 is unable to recognise it on replay. I had no such problems with v1.5.2
Here is the .vd file:

```
sheet   col     row     longname        input   keystrokes      comment
        SqliteSheet     header  set-option      0               
        UsvSheet        delimiter       set-option      ␞               
        UsvSheet        row_delimiter   set-option      ␟               
                        open-file       https://en.wikipedia.org/wiki/List_of_countries_by_income_equality      o       
List_of_countries_by_income_equality            キtable_3       delete-row              d       
List_of_countries_by_income_equality            キtable_4       delete-row              d       
List_of_countries_by_income_equality            キbefore        delete-row              d       
List_of_countries_by_income_equality            キtable_6       delete-row              d       
List_of_countries_by_income_equality            キtable_7       delete-row              d       
List_of_countries_by_income_equality            キafter delete-row              d       
List_of_countries_by_income_equality            キtable_9       delete-row              d       
List_of_countries_by_income_equality            キtable_10      delete-row              d       
List_of_countries_by_income_equality            キtable_11      delete-row              d       
List_of_countries_by_income_equality            キUNandCIA      dive-row                ^J      
UNandCIA        Country_link0           hide-col                -       
UNandCIA        Country_link1           hide-col                -       
UNandCIA        UN R/P_10% [6] _link0           hide-col                -       
UNandCIA        UN R/P_10% [6]          hide-col                -       
UNandCIA        UN R/P_20% [7]          hide-col                -       
UNandCIA        UN R/P_20% [7] _link0           hide-col                -       
UNandCIA        World Bank Gini [3] _% _link0           hide-col                -       
UNandCIA        World Bank Gini [3] _% _link1           hide-col                -       
UNandCIA        CIA R/P [4] _10%                hide-col                -       
UNandCIA        CIA R/P [4] _10%_link0          hide-col                -       
UNandCIA        CIA R/P [4] _Year               hide-col                -       
UNandCIA        CIA R/P [4] _Year_link0         hide-col                -       
UNandCIA        CIA Gini [5] _%         hide-col                -       
UNandCIA        CIA Gini [5] _% _link0          hide-col                -       
UNandCIA        CIA Gini [5] _Year              hide-col                -       
UNandCIA        CIA Gini [5] _Year_link0                hide-col                -       
```

**Comment by anjakefala** [comment]
Confirmed bug @vapniks, thanks for reporting!

**Comment by anjakefala** [comment]
Closing, since this has been fixed on develop since 0704339bae.

The commit will be in the next 2.x release. =)
